packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -254,6 +254,14 @@ This API requires a [My Email Request object](#my-email-request-object) as its r
 
 The requested [My Email object](#my-email-object)
 
+#### Error Responses
+
+If an invalid email is passed to `profile` in the request body, the response will return a 400 BAD REQUEST with error code E0000001.
+
+If the email operation is not enabled for the request `role` on the org, the response will return a 403 FORBIDDEN with error code E0000038.
```

Comment:
Is there a clearer way of saying "email operation"?

Reply:
discussed with Hanyuan offline email operation probably ok..